pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/utils/AuthNaming.java

```diff
@@ -0,0 +1,22 @@
+package edu.harvard.hms.dbmi.avillach.auth.utils;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AuthNaming {
```

Comment:
Why not Enum, and for instance include description for that role here as well ("PIC-SURE admin role for modify users and resources.")?